SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -82,11 +82,15 @@ ESchemaComponentType GetRPCType(UFunction* RemoteFunction)
 
 void USpatialClassInfoManager::CreateClassInfoForClass(UClass* Class)
 {
+	// Remove PIE prefix on class if it exists to properly look up the class.
+	FString ClassPath = Class->GetPathName();
```

Comment:
When do class names have PIE prefixed in them? I assumed PIE was only added for instanced objects.

Reply:
The case I saw this occur was with the level blueprint. When launched with no sublevels loaded, it would have a PIE prefix.